infopark_reactor/lib/reactor/engine.rb

```diff
@@ -1,9 +1,21 @@
-require 'reactor/session_helper'
-
 # -*- encoding : utf-8 -*-
 module Reactor
 
   class Engine < Rails::Engine
+    if ::Rails::VERSION::MAJOR == 5
+      config.after_initialize {
+        if defined?(Obj) && Obj <= RailsConnector::BasicObj
```

Comment:
Puuuh. Ich musste gerade nachschauen, wie `<=>` auf Klassen (hier genauer: Module) [definiert](http://ruby-doc.org/core-2.5.0/Module.html#method-i-3C-3D-3E) ist. Der "Vergleich" zwischen zwei Klassen ist nicht ganz intuitiv, wenn man nicht _weiß_, was `<=>` auf Klassen macht. Vorschlag:

```ruby
if defined?(Obj) && Obj.ancestors.include?(RailsConnector::BasicObj)
```

Das ist ein bisschen mehr verbose, aber gleichzeitig signifikant lesbarer, finde ich.

Reply:
`Obj.ancestors.include?(RailsConnector::BasicObj)` ist equivalent zu `Obj < RailsConnector::BasicObj` aber nicht `<=`.

Und zu den Definitionen von `<` und `<=`: denke daran wie die Klassen in Ruby vererbt werden `class Kid < Parent` 😉

Reply:
Ich lass das jetzt so. Ich musste auch etwas suchen. Aber der Vergleich mach genau das, was ich brauche